pkg/pki/ca/ca.go

```diff
@@ -22,10 +22,29 @@ import (
 	"errors"
 	"time"
 
+	"github.com/golang/glog"
 	"istio.io/auth/pkg/pki"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
I'd be careful in bringing dependency on K8S apis here. Previously CA is CA and all interactions with K8S are delegated to the `controller`.

Reply:
IMHO it's probably ok to directly use it - in future we can refactor when other storage mechanisms are introduced, but too early to abstract it.